### ISSUE TITLE
Card links: border style

### ISSUE THREAD

**Issue by davidhunter08** [nhsuk/nhsapp-frontend#173]
### Description

The current colour contrast of the page background and card links surrounding border is 1.24:1. As it does not meet the [non-text contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) ratio of 3:1 it makes the border somewhat redundant.

<img width="369" alt="Screenshot 2024-10-17 at 10 54 25" src="https://github.com/user-attachments/assets/0c4785e3-b22c-499d-8140-fffe621491a9">

<img width="1479" alt="Screenshot 2024-10-17 at 11 01 56" src="https://github.com/user-attachments/assets/10580d35-23e6-429d-950d-d00b438a8ae9">

## Blog post on non-text contrast of UI elements

https://iknowdavehouse.medium.com/why-debate-non-text-contrast-of-ui-elements-1be4313c9e12

> Every time you design or build something, take 10 minutes to review it with all the parts of the interactive UI elements that do not meet 3:1 contrast ratio removed.
> 
> So if your button background does not meet 3:1 remove it, if your interactive card borders do not meet 3:1 remove them…
> 
> Then ask yourself:
> 
> - Is the interface still understandable?
> - Is the interface still as usable?
> - Could I do more?

**Comment by davidhunter08** [comment]
https://github.com/nhsuk/nhsapp-frontend/issues/11#issuecomment-2063462496

**Comment by davidhunter08** [comment]
## Button v Card link comparison

<img width="758" alt="Screenshot 2024-10-17 at 11 31 35" src="https://github.com/user-attachments/assets/16b7a48f-e891-4396-992d-8a159fd0c6f8">

Something to consider is the visual alignment between buttons and card links (spacing, text size, border width etc).

Current sizes:
- Secondary button border 2px v Card link border 1px
- Button text size (mobile) 16px v Card link text size (mobile) 19px

**Note:** We're currently in the explorative stage of the secondary button and secondary card links.

Also the example above has the divider line connected to the right of the card link, but that still needs to be decided ([see this issue](https://github.com/nhsuk/nhsapp-frontend/issues/172)).

**Comment by michaelgallagher** [comment]
@davidhunter08 re: the above note "Something to consider is the visual alignment between buttons and card links (spacing, text size, border width etc).", i made a little doodle...

<img width="433" alt="card-link-outline-test" src="https://github.com/user-attachments/assets/ea41f512-9ab0-49ef-b7a9-fb2cd6680ef1">

**Comment by davidhunter08** [comment]
Some exploration of border styles, font sizes, spacing on card links:

![card-link-border-style-exploration](https://github.com/user-attachments/assets/a4747bc7-0800-4630-9d4c-d18ea3a885e0)

[Screenshots on Mural](https://app.mural.co/t/nhsdigital8118/m/nhsdigital8118/1698675735058/df034aa9fc31510eed0374c7c6e4851e43c4a4e7?wid=0-1730884277867)

cc @michaelgallagher

**Comment by mariapinillapease** [comment]
**Account Identity management card**

**Context**: In the IA/Nav team we've been exploring creating a new 'identity management card' for the new Account Hub. The aim of this card is to give users information about themselves and given them the option to go and manage health services for others. 
- We've been chatting around making this card visually different to other elements of the page to reinforce this information and set the context of who's account you are on. We've previously heard from users around the importance of this information and that they associate the 'proxy' jump-off to their own personal account information. 

Few exploration screenshots of the visual treatment of the card:
<img width="1019" alt="image" src="https://github.com/user-attachments/assets/bc44c860-8500-441c-89b9-ac29dd03cfe7">
<img width="764" alt="image" src="https://github.com/user-attachments/assets/df1734f8-0a68-40b1-acb2-f5f1366e1c90">
<img width="1060" alt="image" src="https://github.com/user-attachments/assets/86597073-e613-4c35-ab07-24950209ca05">

@michaelgallagher & @davidhunter08 any feedback would be appreciated!

**Comment by davidhunter08** [comment]
Thanks for sharing @mariapinillapease 🙌

IMO tonal colours is the way to go for the 'identity management card' but I'm just not sure on the border styles, it feels quite sharp.

I was actually looking card link styles yesterday afternoon, considering the 'identity management card' alongside the existing card link styles and buttons. I was also thinking about how to make it all feel a bit more 'appy', as we look to go more native 👀

Here's where I got to:

![image](https://github.com/user-attachments/assets/0234daa3-e672-4ce6-bbe9-2b37b29206fd)

![image](https://github.com/user-attachments/assets/2fe720e5-cbee-4280-a0c4-10e8ab90c9f8)

In summary:
- added a 8px border radius (also bumped button radius to 8px from 4px)
- removed the outer borders, reducing the visual noise
- tweaked the chevron and inner border colours to make more versatile for the different colour backgrounds (also reducing visual noise)
- reduced the link font size to 16px, it's currently 18px but the new typographic scale would bump it to 19px, which would be quite large

Next steps:
- review proposed changes with the service manual team to understand how best to progress with visual design changes such as these and how it impacts others components in the design system
- initial accessibility assessment of proposed design (eg: does it work without borders etc)

**Comment by mariapinillapease** [comment]
Thanks @davidhunter08
I like the suggestion of having the tonal colour for the card but without the border. I agree that removing the outer borders reduces the visual noise.
